CMakeLists.txt

```
@@ -12,6 +12,19 @@ set(extern_INSTALL_DIR "${CMAKE_BINARY_DIR}/local")
 include(xml2)
 include(fifo_map)
 
+option(WARNING_AS_ERRORS "Treat any warning as compile time error" OFF)
+option(STATIC_ANALYZERS "Run static analyzers (clang-tidy, cppcheck) during compilation" ON)
```

This mentions several "static analyzers" but ultimately only requires cppcheck. Maybe it would be nicer if it would just try to find cppcheck? I would suggest for the moment to set the default to `OFF`
```suggestion
option(STATIC_ANALYZERS "Run static analyzers (clang-tidy, cppcheck) during compilation" OFF)
```